Dockerfile

```diff
@@ -11,7 +11,7 @@ WORKDIR /usr/src
 COPY . homeassistant/
 RUN \
     pip3 install --no-cache-dir --no-index --only-binary=:all: --find-links "${WHEELS_LINKS}" \
-    -r homeassistant/requirements_all.txt \
+    -r homeassistant/requirements_all.txt -c homeassistant/requirements_all.txt \
```

Comment:
Is it necessary to pass constraints if you're installing that exact file ?

Reply:
Yes :( the wheel build fails currently because pip decide to use not the pinned version from our requirements_all :(